iOS-Email-Client/AsyncTasks/SendMailAsyncTask.swift

```diff
@@ -221,9 +221,14 @@ class SendMailAsyncTask {
                 }
                 return
             }
-            self.updateEmailData(data)
+            let key = self.updateEmailData(data)
             self.handleResponseInMainThread {
```

Comment:
why does this exist?

Reply:
I pass the queue to Alamofire so i expect the completion black to be called in the same background queue. I need to perform updates in the main thread

Reply:
I meant the function, why not call `DispatchQueue.main.async` here? the implementation of `handleResponseInMainThread` does nothing extra

Reply:
ok